plugin/hosts/hostsfile.go

```diff
@@ -117,11 +117,15 @@ func (h *Hostsfile) readHosts() {
 	defer file.Close()
 
 	stat, err := file.Stat()
+	if err != nil {
+		log.Warningf("Failed to get file info: %v", err)
```

Comment:
Determine if the error is empty, just to exit early, so the title need to be  modified.